src/SourceBuild/content/eng/allowed-vmr-binaries.txt

```diff
@@ -38,6 +38,10 @@ src/cecil/symbols/**/Test/Resources/assemblies/*.pdb
 src/cecil/symbols/**/Test/Resources/assemblies/*.dll
 src/cecil/symbols/**/Test/Resources/assemblies/*.mdb
 
+# efcore
+src/efcore/test/EFCore.Sqlite.FunctionalTests/northwind.db # https://github.com/dotnet/source-build/issues/4326
```

Comment:
@ellahathaway - Does the parsing of this file support comments inline with the file pattern?

Reply:
The `allowed-sb-binaries.txt` file has similar inline comments

Reply:
> Does the parsing of this file support comments inline with the file pattern?

It should: https://github.com/dotnet/dotnet/blob/5bdffa4bdc9a7425cf78cad5ba1b86de52dca59b/eng/tools/BinaryToolKit/DetectBinaries.cs#L226.